doc/user/sql-grammar/sql-grammar.bnf

```diff
@@ -432,6 +432,7 @@ table_ref ::=
     | 'LATERAL'? table_func_call
     | '(' join_expr ')'
   ) ('AS'? table_alias ('(' col_alias (',' col_alias)* ')'))?
+table_expr ::= 'TABLE' table_name
```

Comment:
order by and limit?

Reply:
I thought about it! But it seemed exhausting to duplicate all the `ORDER BY` and `LIMIT` syntax and docs for both `TABLE` and `VALUES`. Also `FETCH FIRST` which isn't even documented for `SELECT`. I decided that calling it an "expression" instead of a "statement" was sufficient because the docs are not intending to describe the statement form, but just the expression form. 

That said, we're definitely missing docs on the "set expression" which is defined as `{SELECT ... | TABLE ... | VALUES ...} {UNION | EXCEPT | INTERSECT} {SELECT ... | TABLE ... | VALUES ...} [ORDER BY ...] [LIMIT ...] [FETCH FIRST ...]`. Untangling all that seemed like a large project. 

Since you approved, going to roll with this as is while the build is green, but lmk if you see an easy way to improve things here.